include/sound/sof/topology.h

```diff
@@ -80,7 +80,6 @@ struct sof_ipc_comp_config {
 	struct sof_ipc_cmd_hdr hdr;
 	uint32_t periods_sink;	/**< 0 means variable */
 	uint32_t periods_source;	/**< 0 means variable */
-	uint32_t preload_count;	/**< how many periods to preload */
```

Comment:
make it reserved, you can't change the size of the structure

Reply:
and btw when you change the shared structures we need a matching SOF PR.

Reply:
@plbossart  got it. Will revisit again and resubmit the PRs